src/pages/validators/index.tsx

```diff
@@ -139,26 +138,29 @@ const Validators: React.FC<PropsWithChildren> = () => {
   const [filterValidators, fetchPartialValidator, loading, setLoading] =
     useFetchPartial<IValidator>('validators', 'validator/list', 'name');
 
-  const filters: IFilter[] = [
-    {
-      title: 'Name',
-      data: filterValidators.map(validator => validator.name || ''),
-      onClick: async value => {
-        if (value === 'All') {
-          setQueryAndRouter({}, router);
-        } else {
-          setQueryAndRouter({ name: value }, router);
-        }
-      },
-      onChange: async value => {
-        setLoading(true);
-        await fetchPartialValidator(value);
+  const filters: IFilter[] = useMemo(() => {
```

Comment:
use useMemo only when calculation is expensive
https://react.dev/reference/react/useMemo#skipping-expensive-recalculations